collect_app/build.gradle

```diff
@@ -161,77 +164,79 @@ allprojects {
-    compile group: 'com.jakewharton.timber', name: 'timber', version: '4.5.1'
-    compile group: 'com.android.support', name: 'customtabs', version: '25.3.1'
-    compile group: 'com.android.support', name: 'design', version: '25.3.1'
-    compile group: 'com.google.zxing', name: 'core', version: '3.3.0'
-    compile group: 'com.journeyapps', name: 'zxing-android-embedded', version: '3.5.0'
-    compile group: 'com.android.support', name: 'support-v13', version: '25.3.1'
-    compile group: 'net.danlew', name: 'android.joda', version: '2.9.9'
+    implementation group: 'org.osmdroid', name: 'osmdroid-android', version: '5.6.4'
+    implementation group: 'org.slf4j', name: 'slf4j-android', version: '1.6.1-RC1'
+    implementation group: 'pub.devrel', name: 'easypermissions', version: '0.2.1'
+    implementation(group: 'com.google.api-client', name: 'google-api-client-android', version: '1.22.0') { exclude group: 'org.apache.httpcomponents' }
+    implementation(group: 'com.google.apis', name: 'google-api-services-drive', version: 'v3-rev64-1.22.0') { exclude group: 'org.apache.httpcomponents' }
+    implementation(group: 'com.google.apis', name: 'google-api-services-sheets', version: 'v4-rev463-1.22.0') { exclude group: 'org.apache.httpcomponents' }
+    implementation group: 'com.jakewharton.timber', name: 'timber', version: '4.5.1'
+    implementation group: 'com.google.zxing', name: 'core', version: '3.3.0'
+    implementation group: 'com.journeyapps', name: 'zxing-android-embedded', version: '3.5.0'
+    implementation group: 'net.danlew', name: 'android.joda', version: '2.9.9'
 
     // Real LeakCanary for debug builds only: notifications, analysis, etc
-    debugCompile group: 'com.squareup.leakcanary', name: 'leakcanary-android', version: '1.5.4'
+    debugImplementation group: 'com.squareup.leakcanary', name: 'leakcanary-android', version: '1.5.4'
     // No-Op version of LeakCanary for release builds: no notifications, no analysis, nothing
-    releaseCompile group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
-    testCompile group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
-    androidTestCompile group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
-    odkCollectReleaseCompile group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
+    releaseImplementation group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
+    testImplementation group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
+    androidTestImplementation group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
+    odkCollectReleaseImplementation group: 'com.squareup.leakcanary', name: 'leakcanary-android-no-op', version: '1.5.4'
```

Comment:
I don't like prepending `odk`. My vote is just for `collect`

Reply:
This is a separate change. The point is that this variant is for **ODK**'s Collect, not everyone else's Collect.